variables.tf

```diff
@@ -5,6 +5,9 @@ variable "data_ingest_cidr_block" {}
 variable "peering_cidr_block" {}
 variable "az" {}
 variable "data_landing_bucket" {}
+variable "s3_bucket_name" {}
```

Comment:
can't you auto generate a random name?

Reply:
sure we can but perhaps we wan't an easily readable name?

Reply:
prefix it with something meaningful, see how we do ssm docs https://github.com/UKHomeOffice/dq-tf-ad/blob/master/main.tf#L185